packages/libs/avector/benches/bench.rs

```diff
@@ -0,0 +1,430 @@
+//! Benchmark usecases against:
+//! * (alloc) built-in Vec
```

Comment:
The crate claims to be efficient on read operations but the only operation, which has better performance in these benchmarks, are write-operations in a multi-threaded context. For all other operations, `alloc::Vec` is faster.